raw-svo.defs.lua

```diff
@@ -210,7 +210,10 @@ defs_data = pl.OrderedMap {}
       end
 
       return true
-    end}) -- added in xml w/ conf.gagbreath
+    end,
+    def = [[You are holding your breath.]],
+    ondef = function() return "" end, --workaround to make it show up in defense lists...
```

Comment:
Without having tried this - I think all that would do it make it not show the defence name in the defences list. Do things work if you take it out?